relay-server/src/metrics_extraction/event.rs

```diff
@@ -896,6 +896,51 @@ mod tests {
                     "data": {
                         "db.system": "redis"
                     }
+                },
+                {
+                    "data":{
+                        "device.class":"2",
+                        "environment":"production",
+                        "http.status_code":"200",
```

Comment:
If you want to see this as a tag on the metric, use `http.response.status_code` instead: https://github.com/getsentry/relay/blob/bce54057b84cb1d34bacc8ab91c95d3c7d6be1a2/relay-event-normalization/src/normalize/utils.rs#L15

https://develop.sentry.dev/sdk/performance/span-data-conventions/#http